gradle/libs.versions.toml

```diff
@@ -1,30 +1,29 @@
 [versions]
 androidxCore = "1.13.1"
-androidxLifecycle = "2.8.4"
+androidxLifecycle = "2.8.5"
 kotlinxCoroutinesTest = "1.8.1"
 junit = "4.13.2"
-mockkAndroid = "1.13.9"
+mockkAndroid = "1.13.12"
 jvmTarget = "21"
-kotlin = "2.0.10"
-ksp = "2.0.10-1.0.24"
-gradlePlugin = "8.5.2"
+kotlin = "2.0.20"
+# should match kotlin https://github.com/google/ksp/releases
+ksp = "2.0.20-1.0.25"
+gradlePlugin = "8.6.0"
```

Comment:
Gradle plugin version 8.6.0 seems to require java version 22, which does not match our target compatibility of 21. Also our self-hosted runner seems to be running java_22 which is recommended by this template. So before merging, it seems we should either update the runner to support 22 and change the targetcompatibility to 22...

or revert this gradle version change

Reply:
https://www.notion.so/q42/GitHub-Actions-CI-op-onze-self-hosted-runner-Mac-Mini-7f7d2e6312444b98be521394223db6e2

https://docs.gradle.org/current/userguide/compatibility.html?_ga=2.163671295.1169800436.1726069472-1934319561.1716894485&_gl=1*1353ja0*_gcl_au*MTAxNzIyNTU2MS4xNzI1MjkyOTU5*_ga*MTkzNDMxOTU2MS4xNzE2ODk0NDg1*_ga_7W7NC6YNPT*MTcyNjA2OTQ3Mi41LjEuMTcyNjA2OTQ4Ni40Ni4wLjA.#android

Reply:
With all of the above, I hardly know what I'm talking about

Reply:
Update: even without this change; the self-hosted build fails, this change may not be the cause and could actually be fine https://github.com/Q42/Template.Android/issues/79

Reply:
self-hosted runners: mac-mini pcs somewhere @ Q office that runs builds. It is not enabled by default, that's why the builds are succeeding. See readme